src/__experimental__/components/textbox/textbox.spec.js

```diff
@@ -110,6 +111,22 @@ describe('Textbox', () => {
     });
   });
 
+  it('the required prop is passed to the input', () => {
+    const wrapper = mount(
+      <Textbox
+        value='foo'
+        label='Required'
+        required
+      />
+    );
+
+    const input = wrapper.find('input');
+    expect(input.prop('required')).toBe(true);
+
+    const label = wrapper.find(Label);
+    expect(label.prop('isRequired')).toBe(true);
+  });
```

Comment:
two tests in one case

Reply:
Updated test description because the setup is the same.